client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -68,6 +86,58 @@ export default React.createClass( {
 		return ( this.props.domains || [ this.props.domain ] ).filter( domain => domain.currentUserCanManage );
 	},
 
+	wrongNSMappedDomains() {
+		debug( 'Rendering wrongNSMappedDomains' );
+		const wrongMappedDomains = this.getDomains().filter( domain =>
+			domain.type === domainTypes.MAPPED && ! domain.pointsToWpcom );
+
+		debug( 'Domains:', this.getDomains() );
```

Comment:
Yeah, me too - but the command above produces a bundle with `debug` calls. Maybe there's a step after that that I've missed? But that should be mostly removing comments and whitespace 😕